config.go

```diff
@@ -845,17 +842,17 @@ func (c *Settings) validateAndDefault() error {
 	return nil
 }
 
-// configSection defines a sub-document in the evegreen config
+// ConfigSection defines a sub-document in the evegreen config
 // any config sections must also be added to registry.go
-type configSection interface {
+type ConfigSection interface {
 	// id() returns the ID of the section to be used in the database document and struct tag
 	id() string
-	// get() populates the section from the DB
-	get() error
-	// set() upserts the section document into the DB
-	set() error
-	// validateAndDefault() validates input and sets defaults
-	validateAndDefault() error
+	// Get() populates the section from the DB
+	Get() error
+	// Set() upserts the section document into the DB
+	Set() error
+	// ValidateAndDefault() validates input and sets defaults
+	ValidateAndDefault() error
```

Comment:
Can we just call this Validate? I know we set defaults, but... 

We could also move the default setting to either Get or Set

Reply:
I'd prefer to keep it like this to make it clear that this might change the struct. The base settings struct also has a public Validate method meant to be used to validate the entire document so we would need to change that